pyfunnel/core.py

```diff
@@ -407,10 +412,10 @@ def browse(self, *args, **kwargs):
                 while True:  # Prompt user to retry.
                     inp = input(
                         ('Launching browser yields syslog errors, '
-                         'probably because Chrome is used and the display entered screensaver mode.\n'
-                         'All related processes have been killed by precaution.\n'
-                         'If you have Firefox installed and want to use it persistently, enter p\n'
-                         'Otherwise, do you simply want to retry ([y]/n/p)? '))
+                            'probably because Chrome is used and the display entered screensaver mode.\n'
```

Comment:
Wonder if you want to has the extra spaces (or tabs)?

Reply:
I reformatted the string.